### ISSUE TITLE
Leaf spring suspension attached to the bottom layer, missing free swing

### ISSUE THREAD

**Issue by timmwille** [chopinregis/OS-TINY-HOME-TRAILER#1]
I was noticing the leaf springs are attached to the bottom layer. Maybe I'm missing something, but it looks like there is a degree of freedom missing for them to swing separately up and down without bending the metal flat bars?
https://www.youtube.com/watch?v=BDv8zkL5c6M&t=38

Thx for the great initiative, I'll share this in the Open Source Ecology Network.

**Comment by chopinregis** [comment]
Yes you are right about your observation about the lower part, thought it is not attached, the structural bar is keeping the swing free from motion, so i've been thinking about it aswell and for now i haven't thought of a better solution on how to address that. But nice of your to point it out good friend TWille. Nice video link, I wish i had seen this when i was modelling, very useful.
